randovania/games/dread/json_data/Hanubia.txt

```diff
@@ -865,7 +865,11 @@ Extra - asset_id: collision_camera_012
   > Tunnel to EMMI Zone Exit West
       Any of the following:
           Space Jump or Simple IBJ
-          Speed Booster and Speedbooster Conservation (Beginner)
+          All of the following:
+              Speed Booster
+              Any of the following:
+                  Speedbooster Conservation (Advanced)
+                  Speedbooster Conservation (Beginner) and Use Spin Boost
```

Comment:
Was a dunce and forgot to merge the changes that addressed this into the pull request, this has been reworked to prevent issues with starting in the room, all done based off the conversation had in the discord.